frontend_test.go

```diff
@@ -91,3 +91,67 @@ func TestCasbinJsGetPermissionForUser(t *testing.T) {
 		t.Errorf("Someone can have a non-existing action (rm -rf)")
 	}
 }
+
+func TestCasbinJsGetPermissionForUser2(t *testing.T) {
+	e, err := NewSyncedEnforcer("examples/rbac_model.conf", "examples/rbac_policy.csv")
+	if err != nil {
+		panic(err)
+	}
+	target_str, _ := CasbinJsGetPermissionForUser(e, "alice")
```

Comment:
I think you can drop the rest of the test from here on, if it's just a copy of the other test.

Up until here it makes sure that CasbinJsGetPermissionForUser can be used with a synced enforcer.

Reply:
You are right.